Tests/HummingbirdTests/ApplicationTests.swift

```diff
@@ -866,6 +866,34 @@ final class ApplicationTests: XCTestCase {
             }
         }
     }
+
+    #if compiler(>=6.0)
+    /// Test consumeWithInboundCloseHandler
+    func testConsumeWithInboundHandler() async throws {
+        let router = Router()
+        router.post("streaming") { request, context -> Response in
+            Response(
+                status: .ok,
+                body: .init { writer in
+                    try await request.body.consumeWithInboundCloseHandler { body in
+                        try await writer.write(body)
+                    } onInboundClosed: {
```

Comment:
What's being tested here?

Reply:
That you can run consumeWithInboundCloseHandler and the request body passed in will still be correct.